.gitignore

```diff
@@ -2,6 +2,8 @@
 saml
 .idea/
 .DS_Store
+opt/
+test/PERSIST
```

Comment:
Move "PERSIST" to "/tmp/PERSIST", then you won't need to ignore this directory.

Reply:
Done